ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -346,13 +346,34 @@ router bgp {{ router_bgp.as }}
 {%             if router_bgp.vrfs[vrf].router_id is defined %}
       router-id {{ router_bgp.vrfs[vrf].router_id }}
 {%             endif %}
+{%             if router_bgp.vrfs[vrf].timers is defined %}
+      timers bgp {{ router_bgp.vrfs[vrf].timers }}
+{%             endif %}
 {%             if router_bgp.vrfs[vrf].neighbors is defined and router_bgp.vrfs[vrf].neighbors is not none %}
 {%                 for neighbor in router_bgp.vrfs[vrf].neighbors %}
 {%                      if router_bgp.vrfs[vrf].neighbors[neighbor].remote_as is defined and router_bgp.vrfs[vrf].neighbors[neighbor].remote_as is not none %}
       neighbor {{ neighbor }} remote-as {{ router_bgp.vrfs[vrf].neighbors[neighbor].remote_as }}
 {%                      elif router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is defined and router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is not none %}
       neighbor {{ neighbor }} peer group {{ router_bgp.vrfs[vrf].neighbors[neighbor].peer_group }}
 {%                      endif %}
+{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].description is defined and router_bgp.vrfs[vrf].neighbors[neighbor].description is not none %}
+      neighbor {{ neighbor }} description {{ router_bgp.vrfs[vrf].neighbors[neighbor].description }}
+{%                      endif %}
+{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].next_hop_self is defined and router_bgp.vrfs[vrf].neighbors[neighbor].next_hop_self == true %}
+      neighbor {{ neighbor }} next-hop-self
+{%                      endif %}
+{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].timers is defined and router_bgp.vrfs[vrf].neighbors[neighbor].timers is not none %}
+      neighbor {{ neighbor }} timers {{ router_bgp.vrfs[vrf].neighbors[neighbor].timers }}
+{%                      endif %}
+{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].send_community is defined %}
```

Comment:
`send-community` knob in bgp appears to support many flavors of options, I would recommend implementing this as a simple string, as opposed to looking for a specific configuration.

Reply:
@titom73 @ClausHolbechArista @ankudinov @aphillipps - please let me know what you think?

Reply:
+1 on simplicity, but we need to check if any command variations can be supported as a single line. Otherwise we are better off creating a more generic "nerd-knobs" list which could contain any BGP knob we like. (done that on another project and it is very useful)

Reply:
After discussing with Claus and testing in the lab whatever combination you have it goes as a single string. In this case we proceed with string and update readme the same.